src/__tests__/login.spec.js

```diff
@@ -1,113 +1,144 @@
-import React from 'react';
```

Comment:
i think something isn't right with your prettier

Reply:
What do you mean?